[debug] [2020-05-14T12:08:34.296Z] ----------------------------------------------------------------------
[debug] [2020-05-14T12:08:34.300Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2020-05-14T12:08:34.300Z] CLI Version:   6.8.0
[debug] [2020-05-14T12:08:34.300Z] Platform:      darwin
[debug] [2020-05-14T12:08:34.300Z] Node Version:  v12.14.1
[debug] [2020-05-14T12:08:34.302Z] Time:          Thu May 14 2020 08:08:34 GMT-0400 (Eastern Daylight Time)
[debug] [2020-05-14T12:08:34.302Z] ----------------------------------------------------------------------
[debug] 
[debug] [2020-05-14T12:08:34.321Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-05-14T12:08:34.321Z] > authorizing via signed-in user
[debug] [2020-05-14T12:08:34.321Z] [iam] checking project album-cloud-8c72f for permissions ["firebase.projects.get"]
[debug] [2020-05-14T12:08:34.323Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2020-05-14T12:08:34.323Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2020-05-14T12:08:34.545Z] <<< HTTP RESPONSE 200
[debug] [2020-05-14T12:08:34.566Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/album-cloud-8c72f:testIamPermissions  

[debug] [2020-05-14T12:08:35.329Z] <<< HTTP RESPONSE 200
[debug] [2020-05-14T12:08:35.330Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/album-cloud-8c72f  
 
[debug] [2020-05-14T12:08:35.625Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/ryanpeterson/Web Development/Projects/album-cloud/backend'...
[info] 
[debug] [2020-05-14T12:08:36.674Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.mkdirSync (fs.js:823:3)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9554:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command._action (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
